src/main/scala/dependencies/DependenciesPlugin.scala

```diff
@@ -2,18 +2,93 @@ package dependencies
+      case list => f(list)
+    }
+
+  def createIssueForDep(dep: DependencyUpdate,
+                        issues: Map[String, Issue],
+                        githubClient: GithubClient,
+                        log: Logger): Future[Issue] = {
+    log.info(s"Preparing issue for module `${dep.moduleName}`\n")
+    issues.get(dep.moduleName) match {
+      case Some(issue) =>
+        log.info(s"Found existing open issue (#${issue.number}), updating it\n")
+        githubClient.updateIssue(issue, dep) map { issue =>
+          log.info(s"Issue updated at: ${issue.html_url}")
+          issue
+        }
+      case None =>
+        log.info("Existing issue not found, creating a new one\n")
+        githubClient.createIssue(dep) map { issue =>
+          log.info(s"Issue created at: ${issue.html_url}")
+          issue
+        }
+    }
+  }
+
   lazy val defaultSettings = Seq(
-    dependencyUpdatesExclusions := moduleFilter(organization = "org.scala-lang"),
-    compile <<= (compile in Compile) dependsOn dependencyUpdates
+    showDependencyUpdates := {
+      readUpdates(dependencyUpdatesData.value, streams.value.log) {
+        list =>
+          val fullTable = Seq("Module", "Revision", "Patch", "Minor", "Major") +:
+              list.map(
+                dep =>
+                  Seq(dep.moduleName,
+                      dep.revision,
+                      dep.patch.getOrElse(""),
+                      dep.minor.getOrElse(""),
+                      dep.major.getOrElse("")))
+          streams.value.log.info("\nFound some dependency updates:\n")
+          streams.value.log.info(TablePrinter.format(fullTable))
+          streams.value.log.info("Execute `updateDependencyIssues` to update your issues\n")
+      }
+    },
+    updateDependencyIssues := {
+      readUpdates(dependencyUpdatesData.value, streams.value.log) {
+        list =>
+          streams.value.log.info("Reading GitHub issues\n")
+          sys.props.get("githubToken") match {
+            case Some(accessToken) =>
+              val githubClient = GithubClient(githubOwner.value, githubRepo.value, accessToken)
+              val result = for {
+                issues <- githubClient.findIssuesByModuleName()
+                createdIssues <- Future.sequence(
+                  list map (dep =>
+                              createIssueForDep(dep, issues, githubClient, streams.value.log)))
+              } yield createdIssues
+              result onComplete {
+                case Success(createdIssues) => streams.value.log.info("GitHub issues created or updated\n")
+                case Failure(e) =>
+                  streams.value.log.error(s"Error creating issues")
+                  e.printStackTrace()
+              }
+            case None =>
+              streams.value.log.info(
+                "Can't read the access token, please set the GitHub token with the property 'githubToken' (for ex. `sbt -DgithubToken=XXXXXX`)\n")
```

Comment:
Let's include the link to where to get it and the type of token the user needs in this error msg as well